api/controllers/game/ready.js

```diff
@@ -84,11 +84,8 @@ module.exports = function (req, res) {
             })
             .then(function publish(fullGame) {
               Game.publish([fullGame.id], {
-                verb: 'updated',
-                data: {
-                  change: 'Initialize',
-                  game: fullGame,
-                },
+                change: 'Initialize',
```

Comment:
```suggestion
                change: 'initialize',
```

I think this is causing the test failure. The name was capitalized inconsistently with the other socket events. Let's fix this here